{
  "success": false,
  "error": "Traceback (most recent call last):\n  File \"/usr/local/lib/python3.10/dist-packages/falcon_analyzer-0.2.28-py3.10.egg/falcon/__main__.py\", line 835, in main_impl\n    ) = process_all(filename, args, detector_classes, printer_classes)\n  File \"/usr/local/lib/python3.10/dist-packages/falcon_analyzer-0.2.28-py3.10.egg/falcon/__main__.py\", line 87, in process_all\n    compilations = compile_all(target, **vars(args))\n  File \"/home/im/.local/lib/python3.10/site-packages/crytic_compile/crytic_compile.py\", line 722, in compile_all\n    compilations.append(CryticCompile(target, **kwargs))\n  File \"/home/im/.local/lib/python3.10/site-packages/crytic_compile/crytic_compile.py\", line 211, in __init__\n    self._compile(**kwargs)\n  File \"/home/im/.local/lib/python3.10/site-packages/crytic_compile/crytic_compile.py\", line 633, in _compile\n    self._platform.compile(self, **kwargs)\n  File \"/home/im/.local/lib/python3.10/site-packages/crytic_compile/platform/hardhat.py\", line 183, in compile\n    hardhat_like_parsing(crytic_compile, self._target, build_directory, hardhat_working_dir)\n  File \"/home/im/.local/lib/python3.10/site-packages/crytic_compile/platform/hardhat.py\", line 97, in hardhat_like_parsing\n    path = convert_filename(\n  File \"/home/im/.local/lib/python3.10/site-packages/crytic_compile/utils/naming.py\", line 169, in convert_filename\n    filename = _verify_filename_existence(filename, cwd)\n  File \"/home/im/.local/lib/python3.10/site-packages/crytic_compile/utils/naming.py\", line 119, in _verify_filename_existence\n    raise InvalidCompilation(f\"Unknown file: {filename}\")\ncrytic_compile.platform.exceptions.InvalidCompilation: Unknown file: forge-std/console.sol\n",
  "results": {}
}